### DISCUSSION TITLE
What is the use of subfinder?

### DISCUSSION THREAD

**Discussion by teo-en-ming** [projectdiscovery/subfinder#480] [General]
Good day from Singapore,

May I know what is the use of subfinder?

Regards,

Mr. Turritopsis Dohrnii Teo En Ming
Targeted Individual in Singapore
3rd Jan 2022 Monday

**Comment by olearycrew** [comment]
Subfinder is a passive DNS discovery tool used to discover subdomains that could be in use for the target domain.
